### PR TITLE
#155738443 Create layout file for RecyclerView

### PR DESCRIPTION
**Pull request by bibangamba**
#### What does this PR do?
Adds a recyclerview layout to the project

#### Description of tasks to be done?
* add recyclerview support library to app Gradle dependencies
* add recyclerview layout file and include it in the main_activity layout file

#### How can this be manually tested?
clone the repo and navigate to the branch and check out the layout previews

#### What are the relevant pivotal tracker stories?
[#155738443](https://www.pivotaltracker.com/story/show/155738443)

#### Screenshots
Recyclerview in preview pane
image

Recyclerview item in preview pane
image

### CODE REVIEW

**Comment by joshNic** [comment]
@bibangamba I have noticed you want to merge directly to master could you please not merge to master but instead create a develop branch

**Comment by bibangamba** [comment]
going to change this, forgot to change default branch. thank you for the feedback.

File: app/src/main/res/layout/github_user_list_item.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal"
+    android:padding="5dp">
```
Comment (joshNic) [inline app/src/main/res/layout/github_user_list_item.xml]: @bibangamba  could you please put all the values in the dimens.xml file such that you are only referencing them

**Review by joshNic — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/github_user_list_item.xml (hunk 1 above)
Comment (bibangamba) [reply app/src/main/res/layout/github_user_list_item.xml]: let me do this shortly

Comment (bibangamba) [reply app/src/main/res/layout/github_user_list_item.xml]: implemented this, thanks for the feedback

**Comment by bibangamba** [comment]
`note`: the ci build fail is because this branch doesn't have the config yet

**Review by joshNic — APPROVED** [review]
LGTM

**Pull request merged**